lib/events.js

```diff
@@ -351,7 +351,7 @@ EventEmitter.prototype.prependOnceListener =
       return this;
     };
 
-// emits a 'removeListener' event iff the listener was removed
+// emits a 'removeListener' event if and only if the listener was removed
```

Comment:
(For that matter, it's something that whoever lands the PR can fix up if they want to.)